java/elemental2/dom/integer_entities.txt

```diff
@@ -92,13 +92,16 @@ elemental2.dom.DOMTokenList.length
 elemental2.dom.DOMTokenList.item.index
 elemental2.dom.DOMImplementationList.length
 elemental2.dom.DOMImplementationList.item.index
+elemental2.dom.Element.ALLOW_KEYBOARD_INPUT
 elemental2.dom.Element.childElementCount
 elemental2.dom.Element.clientHeight
 elemental2.dom.Element.clientLeft
 elemental2.dom.Element.clientTop
 elemental2.dom.Element.clientWidth
 elemental2.dom.Element.scrollHeight
 elemental2.dom.Element.scrollWidth
+elemental2.dom.Element.webkitRequestFullScreen.allowKeyboardInput
+elemental2.dom.Element.webkitRequestFullscreen.allowKeyboardInput
```

Comment:
These lines are dupplicates

Reply:
They are different casing on `Screen` versus `screen`. FWIU they are both supported but `webkitRequestFullscreen` was preferred and it was also close to the standard complaint variant `requestFullscreen`

Reply:
You should maybe deprecate the `webkitRequestFullScreen` version